<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox
{
	border:1px solid blank;
}</string>
    </property>
    <property name="title">
     <string>Zookeeper</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Zookeeper(2181)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zkStart">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zkStop">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Zookeeper(2182)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Zookeeper(2183)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zkStart2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zkStart3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zkStop2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zkStop3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QLabel" name="zkMsg">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="zkMsg2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>67</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="zkMsg3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>67</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>401</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox
{
	border:1px solid blank;
}</string>
    </property>
    <property name="title">
     <string>Kafka</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Kafka(9092)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="kfStart">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="kfStop">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Kafka(9093)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Kafka(9094)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="kfStart2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="kfStart3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="kfStop2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="kfStop3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QLabel" name="kfMsg">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="kfMsg2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>67</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="kfMsg3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>67</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>401</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox
{
	border:1px solid blank;
}</string>
    </property>
    <property name="title">
     <string>Topic</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="cParition">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>146</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>72</y>
       <width>58</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Parition:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>111</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Replication:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cReplication">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>111</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>238</width>
       <height>29</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Topic Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cTopic"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="cMsg">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>401</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox
{
	border:1px solid blank;
}</string>
    </property>
    <property name="title">
     <string>Producer </string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>238</width>
       <height>29</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Topic Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="proTopic"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>84</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Message:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="proMessage">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>221</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>270</y>
      <width>681</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox
{
	border:1px solid blank;
}</string>
    </property>
    <property name="title">
     <string>Consumer</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="consumer">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Consumer</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>238</width>
       <height>29</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Topic Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="conTopic"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>84</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Message:</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="consumerMsg">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>551</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>681</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox
{
	border:1px solid blank;
}</string>
    </property>
    <property name="title">
     <string>Topics List</string>
    </property>
    <widget class="QListWidget" name="topicList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>53</y>
       <width>81</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>28</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>List:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Info:</string>
     </property>
    </widget>
    <widget class="QListView" name="info">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>571</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1110</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
